packages/salesforcedx-vscode-lightning/test/vscode-integration/hover.test.ts

```diff
@@ -81,16 +81,27 @@ describe('Aura Hovers', function() {
 
     expect(hoverInstances).to.have.lengthOf.at.least(1);
 
-    const instance = hoverInstances![0];
-
-    expect(instance).to.have.property('contents');
-    expect(instance.contents).to.have.lengthOf.at.least(1);
+    const content = findContentFromInstances(hoverInstances, '**title**');
 
-    const content = instance.contents![0] as MarkdownString;
+    expect(content).not.to.be.undefined;
+    expect(content).not.to.be.null;
 
-    expect(content.value).to.include('title');
-    expect(content.value).to.include(
+    expect(content!.value).to.include('title');
+    expect(content!.value).to.include(
       'The title can include text or another component, and is displayed in the header.'
     );
   });
 });
+
+/** Helper to find the expected hover content
+ *
+ * @param instances - hover instances
+ * @param expectedContent - content that is being searched for
+ */
+function findContentFromInstances(instances: Hover[], expectedContent: string) {
+  for (const instance of instances) {
+    const contents = instance!.contents as MarkdownString[];
+
+    return contents.find(content => content.value.includes(expectedContent));
```

Comment:
not sure I completely understand, but if there is more than once `instance` and it's not in the order you expect, wouldn't this return undefined? Seems like you might want to do instances.find

Reply:
There shouldn't be multiple instances with the content that is being searched for, so I just grab the first content match that is found.

Reply:
maybe I'm missing something, but I meant if the first instance doesn't have the content then it wouldn't make it to the second instance

Reply:
so the for loop here is a little confusing

Reply:
ah. Nice catch.